### PR TITLE
Fixes #199 Clarify online programs info in 'About'

### LINKED ISSUE #199: About page content adjustment for online programs
Adjust about page content to clarify that all students on A2 campus who use Canvas can be found, not all students period. As we have more online programs that don't use Canvas at all this is an important distinction until/unless we can ever get data from the other platforms.

### PR DESCRIPTION
**Pull request by mfldavidson**
Tweaked some language in the about page mostly focused on clarifying that not all A2 campus students are in Student Explorer if they don't use Canvas. Added link to more info on access.

### CODE REVIEW

File: student_explorer/templates/student_explorer/about.html [hunk 1]
```diff
@@ -9,13 +9,12 @@ <h1 class="sub-header">About Student Explorer</h1>
...
-            <a href="https://documentation.its.umich.edu/student-explorer-general">Student Explorer documentation</a>.
+            <a href="https://documentation.its.umich.edu/student-explorer-general" target="_blank">Student Explorer documentation</a>.
         </p>
 
         <h2 class="list-header">Student Explorer Access</h2>
-        <p>Student Explorer is available to all professional staff/faculty advisors at the U-M Ann Arbor campus upon
-            request for access and includes information about all students on the Ann Arbor campus, including graduate
-            students. Students, including peer advisors, are not able to gain access to Student Explorer.
+        <p>Student Explorer is available to all professional staff/faculty advisors at the U-M Ann Arbor campus upon request for access and includes information about all students on the Ann Arbor campus who use Canvas, including graduate
+            students (note: some graduate programs do not use Canvas and those students will not be visible in Student Explorer). Learn more about <a href="https://documentation.its.umich.edu/student-explorer-access" target="_blank">Student Explorer access</a>.
```
Comment (ssciolla) [inline student_explorer/templates/student_explorer/about.html]: Maybe we can link the whole "Learn more ..." expression here, like we did in "About Cohorts"?

**Comment by mfldavidson** [comment]
TBH I didn't think that much about `target="_blank"` I just did it because 1) one link on the page was already set up that way (the one to my M-Community profile), 2) clicking on "Resources & Support" in the header opens a new tab so it felt weird that the same link in the content didn't behave the same way, and 3) users might want to open the documentation to use alongside the existing Student Explorer window as they navigate. However, I read the link you shared as well as some other resources including, of course, [Jakob Nielsen](https://www.nngroup.com/articles/top-10-mistakes-web-design/), and I've decided that I don't have a good enough reason, so I will remove them.

**Comment by mfldavidson** [comment]
I'm going to add one more commit because literally TODAY our request form went live and I may want to link that somewhere to avoid people having to click too many times to get to it. Thinking about that now.

**Comment by mfldavidson** [comment]
Ok I decided it's more pertinent to minimize clicks between "Permission Denied" page and the request form than the "About" page and the request form, so I changed the contact link as I realized you have to be authenticated to submit feedback, and I'm now done and ready for review

**Comment by ssciolla** [comment]
Looking at it now actually.

**Pull request merged**